src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -153,14 +214,26 @@ protected override void Apply(TransactionOperationContext context, BlittableJson
                         }
                         break;
                     case nameof(AddOrUpdateCompareExchangeBatchCommand):
-                        if (cmd.TryGet(nameof(AddOrUpdateCompareExchangeBatchCommand.Commands), out BlittableJsonReaderArray commands) == false)
+                        if (cmd.TryGet(nameof(AddOrUpdateCompareExchangeBatchCommand.Commands), out BlittableJsonReaderArray addCommands) == false)
                         {
                             throw new RachisApplyException($"'{nameof(AddOrUpdateCompareExchangeBatchCommand.Commands)}' is missing in '{nameof(AddOrUpdateCompareExchangeBatchCommand)}'.");
                         }
-                        foreach (BlittableJsonReaderObject command in commands)
+                        foreach (BlittableJsonReaderObject command in addCommands)
                         {
                             Apply(context, command, index, leader, serverStore);
                         }
+                        SetIndexForBackup(context, cmd, index, type);
+                        break;
+                    case nameof(RemoveCompareExchangeBatchCommand):
```

Comment:
Is it possible to unify `AddOrUpdateCompareExchangeBatchCommand` with `RemoveCompareExchangeBatchCommand`?

Reply:
done